src/main/java/frc/robot/Controls.java

```diff
@@ -41,7 +42,9 @@ public class Controls {
   private final Sensors sensors;
   private final SuperStructure superStructure;
 
-  private BranchHeight branchHeight = null;
+  private BranchHeight branchHeight = BranchHeight.LEVEL_FOUR;
+  private ScoringMode scoringMode = ScoringMode.CORAL;
```

Comment:
There are no controls to change this, so the scoring mode is always `CORAL`. At least for testing there will need to be a way to change this to `ALGAE` and back to `CORAL`